<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="18s"
		durationms="17995"
		type="root"
		rid="4a1657484f28efa"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="Fahad Shah"
		rxversion="8.0.0.0"
		host="HUAWEI LUA-U22 (Unknown)
WLAN - 192.168.3.11"
		osversion="-"
		runtimeversion="-"
		procarch="-"
		oslanguage="-"
		dtnlanguage="-"
		screenresolution="-"
		timestamp="2/16/2018 11:23:05 AM"
		timestampiso="2018-02-16T06:23:05"
		endtime="2/16/2018 11:23:23 AM"
		timeoutfactor="1"
		module="TK_Shop">
		<detail>
			TK_Shop
		</detail>
		<varbindings>
		</varbindings>
		<item
			timeRelativeToTestSuiteStartTime="00:17.980"
			timeRelativeToTestModuleStartTime="00:17.995"
			timeWallClock="11:23:23"
			time="00:17.980"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
